decommissioned-site/README.md

```diff
@@ -14,6 +14,14 @@ Example:
 ENV DECOMMISSION_NOTICE="This site has been decommissioned.</br>Find out more at <a href=\"https://www.europeana.eu\">Europeana.eu</a>."
 ```
 
+Additionally the page title & heading can be modified by setting an ENV value for `DECOMMISSION_TITLE`.
+
+Example:
+
+```
+ENV DECOMMISSION_TITLE="Decommissioned | website"
```

Comment:
Won't this result in the default title having " | website" at the end of it?

Reply:
sure, it's mainly a placeholder. I don't think we'll have generic decommissioned pages. 
I guess the `|` is a bit funny in the html. I was mainly thinking about the title in the header.
I'll remove that though, so it'll just say "Decommissioned website" in the generic case.